src/views/catalog/CreateFromInstanceTypes/CreateFromInstanceType.tsx

```diff
@@ -5,20 +5,30 @@ import SelectInstanceTypeSection from '@catalog/CreateFromInstanceTypes/componen
 import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
 import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { convertResourceArrayToMap, getName } from '@kubevirt-utils/resources/shared';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk-internal';
 import { Card, Divider, Grid, GridItem, List } from '@patternfly/react-core';
 
 import AddBootableVolumeButton from './components/AddBootableVolumeButton/AddBootableVolumeButton';
 import BootableVolumeList from './components/BootableVolumeList/BootableVolumeList';
 import CreateVMFooter from './components/CreateVMFooter/CreateVMFooter';
 import SectionListItem from './components/SectionListItem/SectionListItem';
 import useInstanceTypesAndPreferences from './hooks/useInstanceTypesAndPreferences';
-import { INSTANCE_TYPES_SECTIONS } from './utils/constants';
+import {
+  initialInstanceTypeState,
+  INSTANCE_TYPES_SECTIONS,
+  InstanceTypeState,
+} from './utils/constants';
+import { produceVirtualMachine } from './utils/utils';
 
 import './CreateFromInstanceType.scss';
 
 const CreateFromInstanceType: FC<RouteComponentProps<{ ns: string }>> = () => {
   const sectionState = useState<INSTANCE_TYPES_SECTIONS>(INSTANCE_TYPES_SECTIONS.SELECT_VOLUME);
-  const bootableVolumeSelectedState = useState<V1beta1DataSource>();
+  const [bootableVolumeSelected, setBootableVolumeSelected] = useState<V1beta1DataSource>();
```

Comment:
Can we change this to `selectedBootableVolume`? Placing the adjective behind the noun doesn't work in English.

Reply:
sure, sorry for that, fixed